eng/Versions.props

```diff
@@ -110,6 +110,14 @@
     <MicrosoftExtensionsLoggingVersion>$(MicrosoftExtensionsLogging80Version)</MicrosoftExtensionsLoggingVersion>
     <MicrosoftExtensionsLoggingAbstractionsVersion>$(MicrosoftExtensionsLoggingAbstractions80Version)</MicrosoftExtensionsLoggingAbstractionsVersion>
     <MicrosoftExtensionsLoggingConsoleVersion>$(MicrosoftExtensionsLoggingConsole80Version)</MicrosoftExtensionsLoggingConsoleVersion>
+  </PropertyGroup>  
+  <PropertyGroup Label=".NET 9 Dependent" Condition=" '$(TargetFramework)' == 'net9.0' ">
+    <MicrosoftAspNetCoreAuthenticationJwtBearerVersion>$(MicrosoftAspNetCoreApp80Version)</MicrosoftAspNetCoreAuthenticationJwtBearerVersion>
```

Comment:
All of these versions should be referencing a `MicrosoftNETCoreApp90Version` or `MicrosoftAspNetCoreApp90Version` property (note these have `90` instead of `80` in their names).

Those two properties should be referencing `MicrosoftNETCoreAppRuntimewinx64Version` and `MicrosoftAspNetCoreAppRuntimewinx64Version` respectively.

And those two properties need to have their corresponding entries (e.g. https://github.com/dotnet/dotnet-monitor/blob/feature/9.x/eng/Version.Details.xml#L53) to be fixed up to have 9.0 versions that are coherent with the `Microsoft.Dotnet.Sdk.Internal` entry (which also needs to be fixed up to have a 9.0 version).